pkg/config/config.go

```diff
@@ -424,6 +426,9 @@ func (t *tomlConfig) toConfig(c *Config) {
 	c.ImageConfig = t.Crio.Image.ImageConfig
 	c.NetworkConfig = t.Crio.Network.NetworkConfig
 	c.MetricsConfig = t.Crio.Metrics.MetricsConfig
+
+	// Setup SystemContext
+	c.SystemContext.BigFilesTemporaryDir = c.ImageConfig.BigFilesTemporaryDir
```

Comment:
I'm not sure I love handling this here, but I also don't really know the best spot to do so.
@saschagrunert WDYT? systemContext is validated in the RuntimeConfig() validation and not really touched anywhere else. We don't have an ImageConfig validation step, and this isn't really validating, more configuring.

Reply:
probably separate it in a different method and just.
```
c.SystemContext = t.Crio.GetSystemContext(c)
```

Reply:
Yes I would separate it into a dedicated method to expose that we actually change the systemContext. If we have a need to do this later on again we could just point to that method. :+1: